11ty/biblio.ts

```diff
@@ -0,0 +1,35 @@
+import axios from "axios";
+import { readFile } from "fs/promises";
+import { glob } from "glob";
+import uniq from "lodash-es/uniq";
+
+export const biblioPattern = /\[\[([\w-]+)\]\]/g;
```

Comment:
Informative references can be expressed as `[[?lorem]]`. These aren't used often but might need to be aware of the possibility